includes/admin/settings/views/html-settings-tax.php

```diff
@@ -3,273 +3,137 @@
-						<input type="text" placeholder="*" name="tax_rate_country[new-' + size + ']" class="wc_input_country_iso" />\
-					</td>\
-					<td class="state" width="8%">\
-						<input type="text" placeholder="*" name="tax_rate_state[new-' + size + ']" />\
-					</td>\
-					<td class="postcode">\
-						<input type="text" placeholder="*" name="tax_rate_postcode[new-' + size + ']" />\
-					</td>\
-					<td class="city">\
-						<input type="text" placeholder="*" name="tax_rate_city[new-' + size + ']" />\
-					</td>\
-					<td class="rate" width="8%">\
-						<input type="number" step="any" min="0" placeholder="0" name="tax_rate[new-' + size + ']" />\
-					</td>\
-					<td class="name" width="8%">\
-						<input type="text" name="tax_rate_name[new-' + size + ']" />\
-					</td>\
-					<td class="priority" width="8%">\
-						<input type="number" step="1" min="1" value="1" name="tax_rate_priority[new-' + size + ']" />\
-					</td>\
-					<td class="compound" width="8%">\
-						<input type="checkbox" class="checkbox" name="tax_rate_compound[new-' + size + ']" />\
-					</td>\
-					<td class="apply_to_shipping" width="8%">\
-						<input type="checkbox" class="checkbox" name="tax_rate_shipping[new-' + size + ']" checked="checked" />\
-					</td>\
-				</tr>';
-
-			if ( $tbody.find('tr.current').size() > 0 ) {
-				$tbody.find('tr.current').after( code );
-			} else {
-				$tbody.append( code );
-			}
-
-			jQuery( "td.country input" ).autocomplete({
-				source: availableCountries,
-				minLength: 3
-			});
-
-			jQuery( "td.state input" ).autocomplete({
-				source: availableStates,
-				minLength: 3
-			});
-
-			return false;
-		});
-
-		jQuery('.wc_tax_rates td.postcode, .wc_tax_rates td.city').find('input').change(function() {
-			jQuery(this).attr( 'name', jQuery(this).attr( 'data-name' ) );
-		});
-
-		var availableCountries = [<?php
-			$countries = array();
-			foreach ( WC()->countries->get_allowed_countries() as $value => $label )
-				$countries[] = '{ label: "' . esc_attr( $label ) . '", value: "' . $value . '" }';
-			echo implode( ', ', $countries );
-		?>];
-
-		var availableStates = [<?php
-			$countries = array();
-			foreach ( WC()->countries->get_allowed_country_states() as $value => $label )
-				foreach ( $label as $code => $state )
-					$countries[] = '{ label: "' . esc_attr( $state ) . '", value: "' . $code . '" }';
-			echo implode( ', ', $countries );
-		?>];
-
-		jQuery( "td.country input" ).autocomplete({
-			source: availableCountries,
-			minLength: 3
-		});
+<div id="unsaved-changes" style="display:none;">
+	<?php _e( 'There are unsaved changes.', 'woocommerce' ); ?>
+	<pre style="display:none;"></pre>
+</div>
+
+<script type="text/html" id="tmpl-wc-tax-table-row">
+	<tr class="tips" data-tip="<?php echo esc_attr( sprintf( __( 'Tax rate ID: %s', 'woocommerce' ), '{{ data.tax_rate_id }}' ) ); ?>" data-id="{{ data.tax_rate_id }}">
+		<td class="sort">
+			<# if ( ! data.newRow ) { #>
```

Comment:
It was to mimic the old structure of how it used to track additions/removals.  It can probably go away, I was just planning on doing a second pull request to remove the no-longer-used functions from the old method, once the new is merged in.